src/python/pants/backend/python/rules/run_setup_py.py

```diff
@@ -194,44 +199,69 @@ def register_options(cls, register):
            "`--setup-py2-args=\"bdist_wheel --python-tag py36.py37\"`. If unspecified, we just "
            "dump the setup.py chroot."
     )
+    register(
+      '--recursive', type=bool,
```

Comment:
We've generally used `--transitive` for this usecase... is this different in some way?

Reply:
I'll change to `--transitive`, but this is different, in that `--transitive` usually means "the transitive dependency closure of the targets in the original cmd-line specs". But here it means "the owners of those targets", which may not actually be in that closure at all.

Reply:
I'm not certain we should change. Also note that we would need to deprecate the V1 option from `--recursive` to `--transitive` if we make the change here.